tests/Basic/BestPracticesTest.php

```diff
@@ -53,7 +56,7 @@ public static function provideRequiredMethods(): iterable
         ];
     }
 
-    private function assertImplementMethod(string $method, object $object, string $reason = ''): void
+    private function assertMethodIsImplemented(string $method, object $object, string $reason = ''): void
```

Comment:
Is this related to improving PhpStan or just a minor refactoring?

Reply:
Minor refactoring